pandas/core/generic.py

```diff
@@ -6469,7 +6471,12 @@ def bfill(
 
     backfill = bfill
 
-    @doc(klass=_shared_doc_kwargs["klass"])
+    @doc(
+        _shared_docs["replace"],
+        klass=_shared_doc_kwargs["klass"],
```

Comment:
why are you not using the existing pattern, e.g. why are you creating _shared_doc_kwargs?

Reply:
`_shared_doc_kwargs` is used elsewhere in the code base. I must have missed the discussion if it's discouraged.

In the current case, [these](https://github.com/pandas-dev/pandas/pull/38561/files#diff-2882e7769dccf170d039eb30c5a9fd906a9107af9b64a9f95c51919b22b0dbecR669-R676) are needed for both DataFrame and Series, but not for index classes. I figured having these string literals typed in once would be better than having them twice in DataFrame and Series.